docs/commands/status.md

```diff
@@ -20,6 +39,9 @@ A dependency will fail the status checks if:
    - default value: not set, all configured sources
 - `--format`/`-f`: the output format
    - default value: `yaml`
+- `--data-source`/`-d`: where to find the data source of records for status checks
+   - available values: `files`, `configuration`
+   - default value: `files`
```

Comment:
Just a suggestion, I feel like renaming the available value `files` to `cache` would be clearer.

Reply:
😆 that's what I started out calling that option, but I ended up at files because I've gotten feedback in the past that the term `cache` is confusing related to storing files in a repo.  I'm pretty ambivalent myself so if you feel at all strongly about naming this `cache` I'm happy to rename.

Reply:
I don't feel strongly about it either, I just thought that it aligns better with terminology used elsewhere in the project, e.g. the `licensed cache` command.